packages/puppeteer-core/src/cdp/Connection.ts

```diff
@@ -83,15 +83,15 @@ export class Connection extends EventEmitter<CDPSessionEvents> {
   /**
    * @internal
    */
-  get _sessions(): Map<string, CDPSession> {
+  get _sessions(): Map<string, CdpCDPSession> {
     return this.#sessions;
   }
 
   /**
    * @param sessionId - The session id
    * @returns The current CDP session if it exists
    */
-  session(sessionId: string): CDPSession | null {
```

Comment:
I don't think we can change this as its part of the Public API

Reply:
I've reverted _session removal so that `CdpCDPSession` is not exposed.